bin/accessibility-automation/cypress/index.js

```diff
@@ -0,0 +1,113 @@
+/* Event listeners + custom commands for Cypress */
+
+Cypress.on('test:before:run', () => {
+  if (Cypress.env("IS_ACCESSIBILITY_EXTENSION_LOADED") !== "true") return
+  const extensionPath = Cypress.env("ACCESSIBILITY_EXTENSION_PATH")
+
+  if (extensionPath !== undefined) {
+    new Promise((resolve, reject) => {
+      window.parent.addEventListener('A11Y_TAP_STARTED', () => {
+        resolve("A11Y_TAP_STARTED");
+      });
+      const e = new CustomEvent('A11Y_FORCE_START');
+      window.parent.dispatchEvent(e);
+    })
+  }
+})
+
+Cypress.on('test:after:run', (attributes, runnable) => {
+  if (Cypress.env("IS_ACCESSIBILITY_EXTENSION_LOADED") !== "true") return
+  const extensionPath = Cypress.env("ACCESSIBILITY_EXTENSION_PATH")
+  const isHeaded = Cypress.browser.isHeaded;
+  if (isHeaded && extensionPath !== undefined) {
+
+    let shouldScanTestForAccessibility = true;
+    if (Cypress.env("INCLUDE_TAGS_FOR_ACCESSIBILITY") || Cypress.env("EXCLUDE_TAGS_FOR_ACCESSIBILITY")) {
+
+      try {
+        let includeTagArray = Cypress.env("INCLUDE_TAGS_FOR_ACCESSIBILITY").split(";")
+        let excludeTagArray = Cypress.env("EXCLUDE_TAGS_FOR_ACCESSIBILITY").split(";")
+
+        const fullTestName = attributes.title;
+        const excluded = excludeTagArray.some((exclude) => fullTestName.includes(exclude));
+        const included = includeTagArray.length === 0 || includeTags.some((include) => fullTestName.includes(include));
```

Comment:
If `INCLUDE_TAGS_FOR_ACCESSIBILITY` is empty, we run scans for every test?

Reply:
Yes